presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -1564,7 +1564,7 @@ public boolean isFragmentResultCachingEnabled()
         return fragmentResultCachingEnabled;
     }
 
-    @Config("experimental.fragment-result-caching-enabled")
+    @Config("fragment-result-cache.enabled")
```

Comment:
Can you leave the old config on by adding a `LegacyConfig`?  That way people don't need to change their configs when they upgrade.

Reply:
I just added a `LegacyConfig` for the old config.

Reply:
Is it possible to keep the `experimental`, I don't think FRC is a production ready feature

Reply:
But I'm ok, if we have to remove it to make it correct